cmake/cmake_modules/FindPythonLibsNew.cmake

```diff
@@ -0,0 +1,202 @@
+# - Find python libraries
```

Comment:
Please, no vendored and totally outdated cmake stuff.
Pybind11 supports cmake config mode for ages, and does not require any external helpers.

Reply:
See https://github.com/NGSolve/netgen/pull/127